src/stravalib/model.py

```diff
@@ -197,38 +196,6 @@ class DeprecatedSerializableMixin(BaseModel):
     Inherits from the `pydantic.BaseModel` class.
     """
 
-    @classmethod
```

Comment:
I removed this completely. it's really just returning parse_obj anyway.

Reply:
ok that clarification is helpful. I wondered about that because it would become a breaking change if i fully remove from the API